code/modules/admin/verbs/randomverbs.dm

```diff
@@ -874,6 +874,9 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(!check_rights(R_ADMIN))
 		return
 
+	if(alert(usr, "Do you want to [SSshuttle.emergencyNoEscape ? "ALLOW" : "DENY"] shuttle calls?", "Toggle Deny Shuttle", "No", "Yes") != "Yes")
```

Comment:
```suggestion
	if(alert(usr, "Do you want to [SSshuttle.emergencyNoEscape ? "ALLOW" : "DENY"] shuttle calls?", "Toggle Deny Shuttle", "Yes", "No") != "Yes")
```
Yes is always on the left for popups and this will be a pain in the ass otherwise
![image](https://user-images.githubusercontent.com/25063394/93707944-7fd3b300-fb2a-11ea-92c6-25dd344bc65d.png)

Reply:
Oh I always do it the other way around to make sure admins actually know what they are saying yes to but I'll change it